Runtime/Scripts/Schema/MaterialExtension.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
All of these are not used and pull in dependencies uneccessarily. Please remove them!